packages/perseus/src/widgets/deprecated-standin/index.tsx

```diff
@@ -0,0 +1,57 @@
+import Banner from "@khanacademy/wonder-blocks-banner";
+import * as i18n from "@khanacademy/wonder-blocks-i18n";
+import React from "react";
+
+import type {PerseusScore, WidgetExports} from "../../types";
+
+// The props are type `any` on purpose so that this can receive props
+// from any deprecated widget
+type AutoCorrectProps = any;
+type Rubric = any;
```

Comment:
In my heart I still want these to have types, but I also realize we're not actually using anything from them. Doesn't seem worth the effort to type these, but I wonder if it's worth circling back in a separate PR when @jeremywiebe is back or asking in #typescript if there's some TS magic that could work here.